Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Until https://issues.redhat.com/browse/RELEASE-993 is resolved, we need to use :SHA instead of :VERSION tags
+FROM quay.io/redhat-services-prod/app-sre-tenant/er-base-cdktf-aws-main/er-base-cdktf-aws-main:3ab182e AS prod
+
+# keep in sync with pyproject.toml
+LABEL konflux.additional-tags="0.2.0"
+
+# Keep in sync with the 'cdktf-cdktf-provider-random' version in pyproject.toml
+ENV TF_PROVIDER_RANDOM_VERSION="3.6.3"
+ENV TF_PROVIDER_RANDOM_PATH="${TF_PLUGIN_CACHE}/registry.terraform.io/hashicorp/random/${TF_PROVIDER_RANDOM_VERSION}/linux_amd64"
+
+RUN mkdir -p ${TF_PROVIDER_RANDOM_PATH} && \
+    curl -sfL https://releases.hashicorp.com/terraform-provider-random/${TF_PROVIDER_RANDOM_VERSION}/terraform-provider-random_${TF_PROVIDER_RANDOM_VERSION}_linux_amd64.zip \
+    -o /tmp/package-${TF_PROVIDER_RANDOM_VERSION}.zip && \
+    unzip /tmp/package-${TF_PROVIDER_RANDOM_VERSION}.zip -d ${TF_PROVIDER_RANDOM_PATH}/ && \
+    rm /tmp/package-${TF_PROVIDER_RANDOM_VERSION}.zip
+
+# Install dependencies
+COPY pyproject.toml uv.lock ./
+RUN uv sync --no-install-project --no-dev
+
+COPY README.md Makefile cdktf.json validate_plan.py entrypoint.sh ./
+
+# the source code
+COPY er_aws_msk ./er_aws_msk
+
+# Sync the project
+RUN uv sync --no-editable --no-dev
+
+# Empty /tmp to avoid linux permission issues if running this image on OpenShift
+RUN rm -rf /tmp/*
+
+FROM prod AS test
```

Comment:
What is the final image? `prod` or `test` ??

Reply:
This depends. For the test images published to quay.io/redhat-user-workloads, it's the [test stage](https://github.com/app-sre/er-aws-msk/pull/10/files#diff-152e51c390696fa6486bc0219aabbc0fc513229f105440daf8cb158a3931daa4R32). For the prod images published to quay.io/redhat-services-prod it's the [prod stage](https://github.com/app-sre/er-aws-msk/pull/10/files#diff-f7090cf98d4da80f6e57cf8624470a313c6e3b9a9ed3cfaed79149268476cd53R29).

For the local development, it's always the test stage.